spartan/terraform/deploy-aztec-infra/values/archive-resources-prod.yaml

```
@@ -0,0 +1,45 @@
+nodeSelector:
+  local-ssd: "false"
+  node-type: "network"
+  cores: "2"
+
+affinity:
+  podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+            - key: app
+              operator: In
+              values:
+                - node
+        topologyKey: "kubernetes.io/hostname"
+        namespaceSelector: {}
+
+replicaCount: 1
+hostNetwork: true
+
+node:
+  resources:
+    requests:
+      cpu: "3"
+      memory: "12Gi"
```

These requests need to be lower otherwise the pod won't fit on a 2-core machine

```suggestion
      cpu: "1.5"
      memory: "5Gi"
```